src/core_codemods/fix_task_instantiation.py

```diff
@@ -0,0 +1,96 @@
+import libcst as cst
+from typing import Optional
+from core_codemods.api import Metadata, ReviewGuidance, SimpleCodemod, Reference
+from codemodder.codemods.utils_mixin import NameAndAncestorResolutionMixin
+from codemodder.codemods.utils import BaseType, infer_expression_type
+
+
+class FixTaskInstantiation(SimpleCodemod, NameAndAncestorResolutionMixin):
+    metadata = Metadata(
+        name="fix-task-instantiation",
+        summary="Use high-level `asyncio.create_task` API",
+        review_guidance=ReviewGuidance.MERGE_WITHOUT_REVIEW,
```

Comment:
I think I'd like to hedge on this a bit and call it `MERGE_AFTER_CURSORY_REVIEW`. Feel free to push back but I'm exposing my own unfamiliarity with the `async` module.